src/core/connectors.js

```diff
@@ -1497,6 +1497,13 @@ const connectors = [{
 	],
 	js: 'connectors/napster.js',
 	id: 'napster',
+}, {
+	label: 'triplej',
+	matches: [
+		'*://www.abc.net.au/triplej/listen-live/*',
```

Comment:
We could make a more generic connector that covers all radio stations from ABC.

Reply:
Which other ones did you find?

Reply:
Ahhh i see what you mean. I was accessing them all via `/triplej/` but you can access them directly